README.md

```diff
@@ -115,6 +115,57 @@ let collection = account.getCapability(/public/ExampleNFTCollection)
 let ids = collection.getIDs()
 ```
 
+## NFT Metadata
+
+NFT metadata is represented in a flexible and modular way using
+the [standard proposed in FLIP-0636](https://github.com/onflow/flow/blob/master/flips/20210916-nft-metadata.md).
+
+When writing an NFT contract, 
+you should implement the [`Metadata.ViewResolver`](contracts/Metadata.cdc#L3-L6)interface,
+which allows your NFT to implement one or more metadata types called views.
+
+Each `View` represents a different type of metadata, 
+such as an on-chain creator biography or an off-chain video clip.
+
+### Display view
+
+The [`Display`](contracts/Metadata.cdc#L8-L22) view is the most basic metadata view.
+It returns the minimum information required to render an NFT in most applications.
+
+```swift
+import ExampleNFT from "..."
+import Metadata from "..."
```

Comment:
Good flag. So you're saying that contracts named `Metadata` already exist and are in use?

I'd like to avoid using `Views` because it's too generic. How about `MetadataViews`?

Reply:
I think that conctracts have the same namespace as structs and resources and I know that plenty have structs name Metadata. 

MetadataView will work.

Reply:
Yes you're right, structs and resource declarations will override the `Metadata` import.